overview.md

```diff
@@ -2,152 +2,128 @@
 
 ## Overview
 
-Amalgam8 is a multi-tenanted, platform and runtime agnostic, microservices integration framework.
-The microservices in an Amalgam8 application can be polyglot, run in containers, VMs, or even on bare metal.
-
-Amalgam8 provides proxy and registration sidecars and other adapter mechanisms that are used for registering and calling
-microservices. Depending on a microservice's deployment runtime, integration with Amalgam8 can require little or no significant change
-to the microservice implementation code because Amalgam8 is designed to leverage the benefits of any given runtime
-and then provide the easiest possible integration for each. For example, in Kubernetes, a microservice that already has an associated
-Kubernetes service definition can be registered automatically using an Amalgam8 plugin providing
-an adapted view of the Kubernetes services in Amalgam8. 
-Similar plugins are possible for other runtime environments, but where they are not, Amalgam8 also provides registration
-and proxy functionality using sidecars in a number of predefined, but extensible, container images.
-
-In addition to ease of integration, Amalgam8 also provides the integrated microservice-based applications with core functionality
-enabling a great deal of control and testability, addressing some of the most challenging issues faced when moving to
-a microservices-based system. This includes edge and mid-tier version-routing based on traffic percentage, user, and other
-criteria. Delays and failures can also be injected into the path of calls to and between microservices, enabling advanced
-end-to-end resiliency testing. And most importantly, all of this is designed for extensibility and is completely available in open source.
+Amalgam8 is a multi-tenant microservice integration framework that works independently of any particular platform or runtime. The microservices in an Amalgam8 application can be written in any language, and run in Docker containers, virtual machines, or on a bare metal environment.
```

Comment:
integration platform that works independently of any particular container platform or programming language runtime. The microservices running on the Amalgam8 platform can be written in ...